### ISSUE TITLE
installation instructions are outdated

### ISSUE THREAD

**Issue by nbbn** [twostairs/paperwork#251]
Hello, current and working instruction is [here](https://github.com/twostairs/paperwork/blob/master/README.md).

Unfortunately,
  - http://paperwork.rocks/
  - https://github.com/twostairs/paperwork/wiki/Installation---Ubuntu-Debian
  - https://github.com/twostairs/paperwork/wiki/Installation-----CentOS

instructions has nothing about gulp, etc., and following those instructions ends with installation without js/css files.
Please update/delete them or correct me if I'm wrong.
I can fix wiki pages, but please confirm my remarks.

**Comment by crazymao** [comment]
Having same problem, finally got it to run but no css files are loaded.

**Comment by devilx** [comment]
Rebuilt the GitHub page. Now it also says something about gulp. :-) The other two Ubuntu/Centos entries need to still be updated.

**Comment by nbbn** [comment]
I've made debian7 guide (that should work) from ubuntu/debian. No Ubuntu, because there is difference in nodejs installation. No CentOS/ubuntu installation to write and test new tutorials. I will review debian7 tut and try to write centos/ubuntu if anyone won't do it earlier.

**Comment by jamesrascal** [comment]
Added Ubuntu guide, the CentoOS one is still outdated. I am working on updating the CentOS guide now. We will need to add info on installing with alternative databases and configuring SSL and what not. Once the CentOS guide is done I think that this issue can be closed.

**Comment by jamesrascal** [comment]
We now have 3 guides:

- [Debian 7](https://github.com/twostairs/paperwork/wiki/Installing-Paperwork-on-Debian-7)
- [Ubuntu 14.10](https://github.com/twostairs/paperwork/wiki/Installing-Paperwork-on-Ubuntu-14.10)
- [CentOS](https://github.com/twostairs/paperwork/wiki/Installation-----CentOS)

All of them are now updated and tested to be working with the current build of Paperwork.
This issue is now resolved.

**Comment by DeviantEng** [comment]
@jamesrascal Thanks for updating my CentOS install.  I wouldn't have noticed it was outdated otherwise!

**Comment by jamesrascal** [comment]
@DeviantEng Thanks for writing the first guide :+1: saves me time :)

**Comment by ntnn** [comment]
Since the installation for CentOS also applies to Fedora (aside from `dnf`/`yum`) I added that to the wiki.

**Comment by Liongold** [comment]
Is still actual?

**Comment by jamesrascal** [comment]
This can be closed current install instructions are current.

**Comment by Liongold** [comment]
Thanks @jamesrascal
